src/components/Avatar/AvatarMenu.test.js

```diff
@@ -0,0 +1,139 @@
+/* global describe, expect, it, shallow, mount, beforeEach, afterEach  */
+import React from 'react';
+import sinon from 'sinon';
+import * as common from '../../../test/unit/commonTests';
+import AvatarMenu from './AvatarMenu';
+import Avatar from './Avatar';
+
+const sampleData = {
+    name: 'Matt Davies',
+    src: 'https://randomuser.me/api/portraits/lego/2.jpg',
+};
+
+const renderMockMenu = () => shallow((
+    <AvatarMenu avatar={<Avatar name={sampleData.name} src={sampleData.src} />}>
+        <AvatarMenu.Nav />
+        <AvatarMenu.Nav />
+    </AvatarMenu>
+));
+
+describe('AvatarMenu', () => {
+    const sandbox = sinon.sandbox.create();
+
+    beforeEach(() => {
+        sandbox.stub(console, 'error');
+    });
+
+    afterEach(() => {
+        sandbox.restore();
+    });
+
+    common.isConformant(AvatarMenu, {
+        tagName: 'nav',
+        requiredProps: {
+            requiredProps: {
+                name: sampleData.name,
+                src: sampleData.src,
+            },
+        },
+    });
+
+    it('complains if user name is not provided', () => {
+        shallow((
+            <AvatarMenu>
+                <AvatarMenu.Nav />
+                <AvatarMenu.Nav />
+            </AvatarMenu>
+        ));
+        // eslint-disable-next-line no-console
+        expect(console.error).to.be.called();
+    });
+
+    it('complains if children are not provided', () => {
+        shallow(<AvatarMenu name="Matthew Davies" />);
+        // eslint-disable-next-line no-console
+        expect(console.error).to.be.called();
+    });
+
+    it('renders with Avatar', () => {
+        const avatarMenu = renderMockMenu();
+        expect(avatarMenu.find(Avatar).length).to.equal(1);
+    });
+
+    it('default open state is false', () => {
```

Comment:
This is quite picky but I'd prefer if this was 'sets open state to false by default', just so the whole thing reads as a sentence

Reply:
If you like mate, sounds good to me